dascore/utils/namespace.py

```
@@ -31,14 +69,14 @@ def _load_plugin_registry(entry_point_group: str | None) -> dict[str, tuple[str,
     if entry_point_group is None:
-        return {}
+        return FrozenDict()
     # "dascore.patch_namespace" -> "patch"
     stem = entry_point_group.split(".")[-1].replace("_namespace", "")
     csv_path = _PLUGIN_REGISTRY_DIR / f"{stem}.csv"
     if not csv_path.exists():
-        return {}
+        return FrozenDict()
     df = pd.read_csv(csv_path)
-    return dict(zip(df["namespace"], zip(df["package_name"], df["package_url"])))
+    return FrozenDict(zip(df["namespace"], zip(df["package_name"], df["package_url"])))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)